### PR TITLE
Analytics

### PR DESCRIPTION
**Pull request by Kumjami**
#309

### CODE REVIEW

**Comment by nelsonic** [comment]
@Kumjami please assign when ready. 👍

**Comment by Kumjami** [comment]
@nelsonic Please, review. The last checkpoint of #309 will be done in a separate branch, so philip can make a quick review

File: src/components/article/index.js [hunk 1]
```diff
@@ -52,7 +51,7 @@ class ArticleFullPage extends Component {
 
   onAddTagClick () {
     const { articleContent, goBack, addSingleTag } = this.props;
-    addSingleTag(articleContent.name, articleContent.id);
+    addSingleTag(articleContent.name, articleContent.id, undefined, articleContent.name);
```
Comment (nelsonic) [inline src/components/article/index.js]: why `undefined` ?

Comment (Kumjami) [inline src/components/article/index.js]: actually, it could be just false

**Comment by nelsonic** [comment]
Thanks for cleaning that up @Kumjami 👍

**Pull request merged**